test/trip_dispatch_test.rb

```diff
@@ -122,4 +123,40 @@ def build_test_dispatcher
       end
     end
   end
+
+  describe "requesting a trip" do
+    before do
+      @dispatcher = build_test_dispatcher
+      @passenger = RideShare::Passenger.new(id: 1, name: "Smithy", phone_number: "353-533-5334")
+    end
+
+    it "creates a new trip" do
+      new_trip = @dispatcher.request_trip(1)
+
+      expect(new_trip.passenger).must_be_instance_of RideShare::Passenger
+      expect(new_trip.driver).must_be_instance_of RideShare::Driver
+      expect(new_trip).must_be_instance_of RideShare::Trip
+    end
+
+    it "selected an AVAILABLE driver" do
```

Comment:
This test isn't actually related to request_trip method so doesn't really belong in this describe block.